packages/react-google-maps-api/src/useLoadScript.ts

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react'
+
+import { injectScript } from './utils/injectscript'
+import { preventGoogleFonts } from './utils/prevent-google-fonts'
+
+import { isBrowser } from './utils/isbrowser'
+import { LoadScriptProps, defaultLoadScriptProps } from './LoadScript'
+
+export const useLoadScript = ({
```

Comment:
() =>

Reply:
Ah, this is the shame. Babel is actually handling that more properly and transpiles to named function. Sadly TypeScript doesn't bother, so yea, I will change that as well.